README.md

```diff
@@ -1,3 +1,8 @@
+[![Hackage version](https://img.shields.io/hackage/v/hpack.svg?label=Hackage)](http://hackage.haskell.org/package/hpack)
```

Comment:
@andreasabel do you know the reason why this badge is orange instead of say blue?  Is it's because the version is still `v0.*`?  Can we customize this somehow, ideally to match the color of the other badges?

Reply:
I found the docs at https://shields.io/category/dependencies and switch to `informational` color (blue).  Default seems to be `important` (orange).

Reply:
That's easier on the eye for sure :+1:

Reply:
> match the color of the other badges

I think the intention of _green_ is _success_, so if build badges are green, this is quite intuitive.  Informational badges like the hackage version would not change their color, I suppose.